src/StaticLint.jl

```diff
@@ -143,11 +143,11 @@ function followinclude(x, state::State)
     if is_call(x) && length(x) > 0 && isidentifier(x[1]) && valofid(x[1]) == "include"
         path = get_path(x, state)
         if isempty(path)
-        elseif hasfile(state.server, path)
-        elseif isabspath(path) && canloadfile(state.server, path)
-            loadfile(state.server, path)
-        elseif hasfile(state.server, joinpath(dirname(getpath(state.file)), path))
-            path = joinpath(dirname(getpath(state.file)), path)
+        elseif isabspath(path)
+            if hasfile(state.server, path)
+            elseif isabspath(path) && canloadfile(state.server, path)
```

Comment:
```suggestion
            elseif canloadfile(state.server, path)
```
We already check for that in the condition above.